bower.json

```diff
@@ -16,11 +16,14 @@
     "output"
   ],
   "dependencies": {
-    "purescript-dom": "^3.1.0",
-    "purescript-aff": "^2.0.1",
-    "purescript-arraybuffer-types": "^0.2.0"
+    "purescript-dom": "^4.0.0",
+    "purescript-aff": "^3.0.0",
+    "purescript-arraybuffer-types": "^2.0.0"
   },
   "devDependencies": {
-    "purescript-console": "^2.0.0"
+    "purescript-console": "^3.0.0"
+  },
+  "resolutions": {
+    "purescript-arraybuffer-types": "^2.0.0"
```

Comment:
Is this actually required? I wouldn't expect to publish a library with any resolutions section